app/views/pulls/show.html.erb

```diff
@@ -2,6 +2,10 @@
 
 <h2><%= l(:label_pull_request) %> #<%= @pull.id %></h2>
 
+<% if @pull.broken? %>
+  <p class="warning"><span class="icon icon-lock"><%= l(:text_pull_broken) %></span></p>
```

Comment:
I question your choice of the icon (lock) in relation to the type of message. You should maybe use a warning or broken link kind of icon OR use an "alert" type of message.

Reply:
Done.